File: java/src/quiz/backjoon/backjoon_1_11.java
package quiz.backjoon;

import java.util.Scanner;

public class backjoon_1_11 {


    public static void main(String[] args) {

        // 입력형식
        Scanner sc_1 = new Scanner(System.in);
        String input_1 = sc_1.nextLine();
        int A = Integer.valueOf(input_1);

        Scanner sc_2 = new Scanner(System.in);
        String input_2 = sc_2.nextLine();
        int B = Integer.valueOf(input_2);

        // 두 숫자 모두 세자리

        int line_1 = A * ((B % 100) % 10);
        int line_2 = A * ((B % 100) / 10 );
        int line_3 = A * (B / 100);
        int line_4 = A * B;

        System.out.println(line_1);
        System.out.println(line_2);
        System.out.println(line_3);
        System.out.println(line_4);


    }

}

File: 2.kotlin/src/backjoon/backjoon_1_11.kt
package backjoon


fun main(args: Array<String>) {

//    var sc: Scanner = Scanner(System.`in`)
//    var book_kotlin_200.input = sc.nextLine()

    // 첫째 줄에 (A+B)%C, 둘째 줄에 (A%C + B%C)%C, 셋째 줄에 (A×B)%C, 넷째 줄에 (A%C × B%C)%C를 출력한다.

    var sc: String? = readLine()
    var input: String = sc.toString()

    var splitInput = input.split(" ")
    var arr = Array<Int>(100, { 0 })

    var count: Int = 0
    for (value in splitInput) {
        arr[count++] = value.toInt()
    }

    var A: Int = arr[0]
    var B: Int = arr[1]
    var C: Int = arr[2]

    var line_1 = (A + B) % C
    var line_2 = (A % C + B % C) % C
    var line_3 = (A * B) % C
    var line_4 = (A % C * B % C) % C

    println(line_1)
    println(line_2)
    println(line_3)
    println(line_4)

}